components/style/list_properties.py

```diff
@@ -10,9 +10,14 @@
 
 style = os.path.dirname(__file__)
 sys.path.insert(0, os.path.join(style, "Mako-0.9.1.zip"))
+
+from mako.lookup import TemplateLookup
 from mako.template import Template
 
-template = Template(filename=os.path.join(style, "properties.mako.rs"), input_encoding='utf8')
+lookup = TemplateLookup(directories=[style])
+template = Template(filename=os.path.join(style, "properties.mako.rs"),
+                    input_encoding='utf8',
+                    lookup=lookup)
```

Comment:
These changes are needed to be able to use %include. It was significantly harder to be able to use it from `list_properties.py`, so in that sense your (Simon) change in the other branch where you refactor `list_properties.py` away is kind of nice.